components/scream/src/share/field/field_manager.hpp

```diff
@@ -69,6 +70,11 @@ class FieldManager {
   void registration_ends ();
   void clean_up ();
 
+  // Adds a pre-existing field to the FieldManager.
```

Comment:
Maybe "Adds a field to be managed by the FieldManager, that can be used by atmospheric processes." I was confused to read that the "pre-existing" field must not exist already. :-)